src/types/fetch.ts

```diff
@@ -3,6 +3,11 @@ import type { FetchRequest, FetchOptions, FetchResponse } from 'ohmyfetch'
 // An interface to extend in a local project
 export interface InternalApi { }
 
+export type NitroFetchRequest =
+Exclude<keyof InternalApi, '/__nuxt_error'>
+| (`${string}${'/'|'.'}${string}` & {})
+| Exclude<FetchRequest, string>
```

Comment:
We should avoid referencing Nuxt internals here. (The solution will be to make this path internal-only in Nuxt and avoid adding it to InternalApi.)

```suggestion
export type NitroFetchRequest = keyof InternalApi | (`${string}${'/'|'.'}${string}` & {}) | Exclude<FetchRequest, string>
```

Reply:
Right! fixed. sorry I totally missed that, silly mistake 😭